### PR TITLE
Tutorials split

### PR DESCRIPTION
**Pull request by bulaienTang**
Split into three notebooks:
Data_Processing.ipynb
Model_Training_&_Evaluation.ipynb
Visualization.ipynb

### CODE REVIEW

File: src/climate_learn/utils/visualize.py [hunk 1]
```diff
@@ -109,6 +118,11 @@ def visualize_mean_bias(model_module, data_module, save_dir=None):
         x, y, _, _ = batch  # B, 1, 32, 64
         x = x.to(model_module.device)
         y = y.to(model_module.device)
+        # print(x.shape, y.shape)
```
Comment (jasonjewik) [inline src/climate_learn/utils/visualize.py:121]: Remove this comment.

File: src/climate_learn/utils/visualize.py [hunk 2]
```diff
@@ -109,6 +118,11 @@ def visualize_mean_bias(model_module, data_module, save_dir=None):
...
+        if len(x.shape) == 5: 
+            x = x.squeeze(1)
+        x = interpolate_input(x, y)
+        # print(x.shape)
```
Comment (jasonjewik) [inline src/climate_learn/utils/visualize.py:125]: Remove this comment.

**Review by jasonjewik — CHANGES_REQUESTED** [review]
Please do the following:
- [x] Delete `docs/.DS_Store`
- [ ] Format your code by running `black .` at the top-level directory

**Comment by jasonjewik** [comment]
IMO, the individual feature notebooks should not have as much text. Someone should be able to look at the notebook and understand how to run the code in less than a minute. If they want more details, we can provide that in our documentation or point them to the bigger tutorial notebook. @aditya-grover do you have thoughts on what the tutorial notebooks should look like?

**Comment by bulaienTang** [comment]
@jasonjewik Please take a look, I made these notebooks more concise

**Comment by aditya-grover** [comment]
Thanks, Bryan. I was also wondering if we should we have task-specific tutorial notebooks: one for weather forecasting, one for downscaling. I guess it could be part of a different pull request.

**Comment by bulaienTang** [comment]
No problem, I can create a new PR for task-specific notebooks later. Should we close this one and merge it into the main branch?

**Comment by aditya-grover** [comment]
Sure. Will let @jasonjewik make the final approval.

**Comment by jasonjewik** [comment]
lgtm

**Comment by jasonjewik** [comment]
In a near-future commit, we should include links to run these notebooks somewhere. Perhaps we can host them directly in the docs, or on Google Colab.

**Comment by prakhar6sharma** [comment]
I know @bulaienTang will work on individual notebooks later but just wanted to point out that currently some of the notebooks don't work for me. I think it is caused because while importing sometimes the import path that we are using is `src.climate_learn` instead of `climate_learn`.  I think for all these tutorial notebooks, we should be consistent.

**Comment by jasonjewik** [comment]
@prakhar6sharma yes, the iterdatamodule notebook doesn't work at the moment. It's not really a tutorial notebook, it's just there because I wanted to save a code example of how to use the IterDataModule, but I haven't found a proper place to put it yet.

**Pull request merged**